app/views/content/shared/quotes/_quote_progression_aaron.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= render Content::QuoteComponent.new(
+  text: "The continued professional development I've received throughout my career so far has shaped me. It's equipped me with a range of skills that enable me to be a better version of myself and subsequently lead to career progression.",
```

Comment:
```suggestion
  text: "The continued professional development I've received throughout my career so far has shaped me. It's equipped me with a range of skills that enable me to be a better version of myself and has subsequently led to career progression.",
```

I find this end of this quote slightly hard to read - does it need tweaking slightly? But I may have misunderstood the context

Reply:
```suggestion
  text: "The continued professional development I've received throughout my career so far has shaped me. It's equipped me with a range of skills that enabled me to be a better version of myself and subsequently led to career progression.",
```